tests/pamiq_core/console/test_cui.py

```diff
@@ -0,0 +1,212 @@
+import json
+import re
+import sys
+from collections.abc import Generator
+from unittest.mock import MagicMock, patch
+
+import httpx
+import pytest
+
+from pamiq_core.console.cui import Console, main
+
+
+class TestConsole:
+    @pytest.fixture
+    def mock_httpx(self) -> Generator[MagicMock, None, None]:
+        # Mock httpx module imported in pamiq_core/console/cui.py
+        with patch("pamiq_core.console.cui.httpx") as mock_httpx:
+            # Mock GET response
+            mock_response = MagicMock()
+            mock_response.text = json.dumps({"status": "running"})
+            mock_httpx.get.return_value = mock_response
+            # Mock POST response
+            api_response = MagicMock()
+            api_response.text = json.dumps({"result": "success"})
+            mock_httpx.post.return_value = api_response
+            yield mock_httpx
+
+    @pytest.fixture
+    def console(self, mock_httpx: MagicMock) -> Console:
```

Comment:
mockオブジェクトは型アノテーションしなくても良い。モックであることは`mock_` prefixで判別する方式にするし、型チェックもあまり意味をなさないし、エディタの補完も基本効かないので。
```suggestion
    def console(self, mock_httpx) -> Console:
```

Reply:
https://github.com/MLShukai/pamiq-core/pull/210/commits/05ddd71f1ab70ffe25083e45abecceaa31c541ca でmockオブジェクトの型を削除